content/en/docs/concepts/services-networking/service-traffic-policy.md

```diff
@@ -9,7 +9,7 @@ weight: 45
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.23" state="beta" >}}
```

Comment:
You may want to update line 23-24 accordingly since the gate is enabled by default.

Reply:
@tengqm i have made changes, now you can check